templates/documents/general_queue.php

```diff
@@ -12,19 +12,19 @@ function submit_documents()
 
 <a href="controller.php?practice_settings&<?php echo $this->top_action; ?>document_category&action=list" 
    onclick="top.restoreSession()" class="css_button" >
-<span>Edit Categories</span></a><a href="#" onclick="submit_documents();" class="css_button" target="_self" ><span>Update files</span></a>
-<input type="hidden" name="process" value="<?php echo self::PROCESS;?>" /><br><br>
+<span><?php echo xl("Edit Categories");?></span></a><a href="#" onclick="submit_documents();" class="css_button" target="_self" ><span>Update files</span></a>
```

Comment:
Alright! I had seen `library/translation.inc.php` , so thought of using `xl(string)`. Can you elaborate on some differences between xl and xlt or maybe a link to docs would also be enough.